vmdk_plugin/E2E_Tests/neg_verifyTenantCreation_test.go

```diff
@@ -0,0 +1,104 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test covers two negative scenarios:
+// 1. To verify that we are not able to add a non-existent vm to a tenant.
+// 2. To verify that we are not able to create a new tenant with the same name as an existing tenant.
+
+// Test will ssh to the esx and  try creating a tenant by passing non-existent vm
+// with the tenant creation command. This operation will not succeed.
+// Test will then try creating two tenants with the same name.
+// We cannot have two tenants with the same name..
+
+// Test assumes that SSH cert has been setup to enable password-less login
+
+package e2e_test
+
+import (
+	TestUtil "github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/test_util"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+)
+
+var (
+	host             string = os.Getenv("ESX")
+	tenantName       string = ""
+	ADMIN_CLI_TENANT string = "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant "
+	vmName           string = ""
+)
+
+/*
+Test steps:
+1. Create a new tenant named by passing a non-existent vm.
+   Command: /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant create --name=<tenant_name> --vm-list=<vmName>
+2. Do a list of tenants and verify that tenant creation failed.
+   Command: /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls
+3. Create a new tenant.
+   Command: /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant create --name=<tenantName>
+4. Try creating another tenant with same name.
+5. Do a list of tenants and check if we have only one with same name  present.
+*/
+
+func TestVerifyTenantCreation(t *testing.T) {
+	log.Println("This is a negative test to verify that tenant creation by passing non-existent vm fails." +
+		" Test also verifies that tenant creation fails if another tenant with the same name already exists.")
+	checkAddingNonEsxtntVm(t)
+	checkAddingSameNameTenant(t)
+	testCleanup()
+}
+
+// This method verifies that we cannot create a tenant
+// if we pass non-existent vm to tenant creation command
+func checkAddingNonEsxtntVm(t *testing.T) {
+	tenantName = "tenant1_" + strconv.FormatInt(time.Now().Unix(), 20)
```

Comment:
let's stick with the approach used for creating volume name with random string ... it is better to update/rename existing getVolumeNameRandomly to maintain consistency